src/resource/auth/auth.service.ts

```diff
@@ -89,4 +93,18 @@ export class AuthService {
     }
     return true;
   }
+
+  async resetPassword(_resetDetail: ResetPasswordInput) {
+    const { resetToken, password } = _resetDetail;
```

Comment:
Since we not using `_resetDetail` in this method, maybe we can just deconstruct the `resetToken, password` from the function parameter?